stdlib/3/tokenize.pyi

```diff
@@ -6,9 +6,9 @@ from typing import Any, Callable, Generator, Iterable, List, NamedTuple, Optiona
 from builtins import open as _builtin_open
 from token import *  # noqa: F403
 
-COMMENT = 0
-NL = 0
-ENCODING = 0
+COMMENT = ...  # type: int
+NL = ...  # type: int
+ENCODING = ...  # type: int
```

Comment:
You'll want to update token.py too in that case; I followed the style from that stub here.